src/react-components/rtc-debug-panel.js

```diff
@@ -0,0 +1,1011 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import { AreaChart, Area, ResponsiveContainer } from "recharts";
+import { FormattedMessage } from "react-intl";
+import styles from "../assets/stylesheets/rtc-debug-panel.scss";
+
+const TransportType = {
+  SEND: "send_transport",
+  RECEIVE: "receive_transport"
+};
+
+const StatsType = {
+  INBOUND_RTP: "inbound-rtp",
+  OUTBOUND_RTP: "outbound-rtp",
+  LOCAL_CANDIDATE: "local-candidate",
+  REMOTE_CANDIDATE: "remote-candidate",
+  CANDIDATE_PAIR: "candidate-pair"
+};
+
+const STATS_REFRESH_TIME = 500;
+const PRODUCERS_KEY = "producers";
+const CONSUMERS_KEY = "consumers";
+const MEDIASOUP_DOC_BASE_URL = "https://mediasoup.org/documentation/v3/libmediasoupclient/api/";
```

Comment:
Nice touch with the built in documentation links

Reply:
At least will save some links for most users.